.github/dependabot.yml

```diff
@@ -28,6 +28,8 @@ updates:
     - neuro-inc/mlops
   allow:
     - dependency-type: "all"
+  ignore:
+    - dependency-name: "torch*"
   registries:
     - pytorch
```

Comment:
We probably could delete the Pytorch registry from this config, wdyt?

Reply:
I was thinking about it, but decided to keep in case we unignore pytorch in the future: I imagine remembering a separate registry is required would be a problem :)
If you think we don't need it, I happy to remove.

Reply:
You're right, let's keep it for the future.